service_thing_aggregate/app/controllers/HealthCheckController.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ *  MUSIT is a museum database to archive natural and cultural history data.
+ *  Copyright (C) 2016  MUSIT Norway, part of www.uio.no (University of Oslo)
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License,
+ *  or any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+package controllers
+
+import com.google.inject.Inject
+import no.uio.musit.healthcheck.HealthCheckService
+import play.api.libs.json.Json
+import play.api.mvc.{Action, Controller}
+
+import scala.concurrent.ExecutionContext
+
+class HealthCheckController @Inject() (
+    healthCheckService: HealthCheckService
+)(
+    implicit
+    ec: ExecutionContext
+) extends Controller {
+
+  def healthCheck =
```

Comment:
Formatting. Slightly undocumented this one. But we try to keep the `Action` statement next to the equals operator throughout the Controllers in our codebase.

```scala
def healtCheck = Action.async { implicit request =>
  ???
}
```
```